Model/IO/Import/BdfParser.cpp

```diff
@@ -0,0 +1,96 @@
+#include "BdfParser.h"
+
+using MeshIS::Model::Common::Vertex;
+using MeshIS::Model::Common::VertexID;
+using MeshIS::Model::Common::Element_T3;
+using MeshIS::Model::Common::Element_Q4;
+using MeshIS::Model::Common::Element_T4;
+using MeshIS::Model::Common::Element_P6;
+using std::move;
+using std::stoi;
+using std::stod;
+
+void BdfParser::parse_grid_point(stringstream & stream, CMR & mesh)
+{
+    stream.ignore(2 * COLUMN_LENGTH);
+
+    Vertex vertex;
+    for (auto& coordinate : vertex) {
+        const string column = convert_chunk_to_string(stream);
+        coordinate = stod(column);
+    }
+
+    mesh.vertices.push_back(move(vertex));
+}
+
+void BdfParser::parse_triangular_face(stringstream & stream, CMR & mesh)
+{
+    stream.ignore(2 * COLUMN_LENGTH);
+
+    Element_T3 element;
+    for (auto& vertex_id : element) {
+        const string column = convert_chunk_to_string(stream);
+        vertex_id = stoi(column);
+    }
+
+    chrom::decrement_values(element);
+    mesh.elementsT3.push_back(move(element));
+}
+
+void BdfParser::parse_quadrilateral_face(stringstream & stream, CMR & mesh)
+{
+    stream.ignore(2 * COLUMN_LENGTH);
+
+    Element_Q4 element;
+    for (auto& vertex_id : element) {
+        const string column = convert_chunk_to_string(stream);
+        vertex_id = stoi(column);
+    }
+
+    chrom::decrement_values(element);
+    mesh.elementsQ4.push_back(move(element));
+}
+
+void BdfParser::parse_tetra_volume(stringstream & stream, CMR & mesh)
```

Comment:
Unfortunely, element types that I encountered are not the same across the .bdf and .nas formats. So in order to create uniform interface, I've made few functions parsing elements part of interface even though I haven't spotted them in the file but it's obvious that they exist. What should I do, mark the as "= delete" until implemented?